pool/app/changelog/entity.ml

```diff
@@ -0,0 +1,85 @@
+open Ppx_yojson_conv_lib.Yojson_conv
+
+exception Exception of string
+
+module Id = struct
+  include Pool_common.Id
+end
+
+module Change = struct
+  type t = Yojson.Safe.t * Yojson.Safe.t [@@deriving eq, show]
+
+  let t_of_yojson (json : Yojson.Safe.t) : t =
+    match json with
+    | `Tuple [ a; b ] -> a, b
+    | _ -> raise (Exception "TODO")
+  ;;
+
+  let yojson_of_t (a, b) = `Tuple [ a; b ]
+end
+
+module Changes = struct
+  (* TODO: There has to be a better way *)
```

Comment:
Add the link to the comment. Currently, the `@to_yojson/@of_joyson` is working for record types but not for variants.

[Documentation `@to_yojson/@of_joyson`](<https://github.com/ocaml-ppx/ppx_deriving_yojson?tab=readme-ov-file#to_yojson--of_yojson>)